src/robot_dart/gui/magnum/gs/camera.hpp

```diff
@@ -52,6 +52,7 @@ namespace robot_dart {
                     Magnum::Float fov() const { return static_cast<Magnum::Float>(_fov); }
                     Magnum::Int width() const { return _camera->viewport()[0]; }
                     Magnum::Int height() const { return _camera->viewport()[1]; }
+                    Magnum::Matrix3 intrinsicMatrix();
```

Comment:
This should have the following signature: `Magnum::Matrix3 const intrinsic_matrix()` to be consistent with the style.